readme.md

```diff
@@ -57,6 +57,20 @@ If you want to save the image file, you can use the saveImage method instead.
 $builder->saveImage('(10)123456(400)11');
 ```
 
+If you want to disable force defined width image, use false on optional second parameter (default:true).
+If you resize image on print, label will be distorted.
+
+```php
+$builder->setWidth(500,false);
+```
+
+If you want to disable label to use your own or none.

```

Comment:
I dont thing it works the way you described at the moment ;)

Reply:
Its working on test i do!
```
public function setWidth($width, $forceWidth = true)
    {
        $this->width = $width;
        $this->forceWidth = $forceWidth;

        return $this;
    }
```
Text exposed talk about this:
`$builder->setBarcodeLabel(false);`

Reply:
but it says I may use my own or none. I am sure this fork for latter case ;)

Reply:
Sorry, im not native English . Im talking about your own method, like print img + html label.
Feel free to describe it as you want.

Reply:
I am not native neither ;) All I mean, I would be nice to be able to introduce own label. It could of course be just an empty label. It is how I understood your comment. I like your attention to details, literally the fact you bothered to update readme. Good job.